docs/blog/posts/2023-07-17-no-unlimited-free-usage.md

```diff
@@ -0,0 +1,65 @@
+---
+draft: false
+date: 2023-07-17
+search:
+  exclude: true
+authors:
+  - fkorotkov
+categories:
+  - announcement
+---
+
+# Limiting free usage of Cirrus CI
+
+Unfortunately the day has come. As a self-bootstrapped company Cirrus Labs can no longer provide unlimited usage of Cirrus CI
+for public repositories for free. We have to put a limit on the amount of compute resources that can be consumed for free each month
+by organizations and users. Starting September 1st 2023, there will be an upper monthly limit on free usage equal to 40 compute credits
+(equal to 10,000 CPU-minutes for Linux tasks).
+
+The reason for the change is that we want to continue being a profitable business and keep Cirrus CI running,
+but unfortunately we haven’t found a better solution for a couple of ongoing issues described below.
+
+<!-- more -->
+
+**Crypto miners are still active.** Methodically, a lot of CI vendors one after another were restricting free usage because of this single reason.
```

Comment:
```suggestion
**Crypto miners are still active.** Methodically, a lot of CI vendors one after another were restricting free usage because of this [single reason](https://webapp.io/blog/crypto-miners-are-killing-free-ci/).
```